tools/boxstarter/node_boxstarter

```diff
@@ -0,0 +1,21 @@
+# Boxstarter (http://boxstarter.org/) script for Node.js prerequisites
+#
+# To install either open this link in IE or Edge:
+#  http://boxstarter.org/package/nr/url?https://raw.githubusercontent.com/nodejs/node/master/tools/boxstarter/node_boxstarter
+#
+# Or run those commands in a Powershell terminal:
+#  iex ((New-Object System.Net.WebClient).DownloadString('http://boxstarter.org/bootstrapper.ps1'))
+#  get-boxstarter -Force
+#  Install-BoxstarterPackage https://raw.githubusercontent.com/nodejs/node/master/tools/boxstarter/node_boxstarter -DisableReboots
+#
+# For more detail see
+# https://github.com/nodejs/node/blob/master/tools/boxstarter/README.md
+#
+
+# Git and Unix tools will be added to the PATH
+choco install git -params /GitAndUnixToolsOnPath -y
+choco install python2 -y
+
+# Installs VS 2017 Build Tools
+choco install visualstudio2017buildtools -y
+choco install visualstudio2017-workload-vctools -y
```

Comment:
What is the benefit of using Boxstarter here rather than just using chocolatey directly? Is it the web url? 

Looking at [their page](http://boxstarter.org/WhyBoxstarter), I'm not seeing anything obviously beneficial, and it's another thing for new starters to understand.

```console
:: Copied from chocolatey website
@"%SystemRoot%\System32\WindowsPowerShell\v1.0\powershell.exe" -NoProfile -InputFormat None -ExecutionPolicy Bypass -Command "iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))" && SET "PATH=%PATH%;%ALLUSERSPROFILE%\chocolatey\bin"

choco install git -params /GitAndUnixToolsOnPath -y
choco install python2 visualstudio2017buildtools visualstudio2017-workload-vctools -y
```

Reply:
Maybe use a more generic directory name like `tools/bootstrap/` and then iterate over recipes.

Reply:
This is an improvement either way, but I'd go with boxstarter rather than just chocolatey. For users that want to understand this or already know about chocolatey, the script is very easy to understand and use as a guide for their own custom installation (plus, dependencies are already documented in `BUILDING.md`).

Looking at the Boxstarter page, it adds a lot of resilience to the process (reboots, disabling updates, etc), moving it much closer to the ideal one-click-no-issues setup.